text/entities/0264-resource-and-entities.md

```diff
@@ -0,0 +1,799 @@
+- **Entity Detectors (new)**: Detecting an entity that is relevant to the current instance of the SDK.  For example, this would detect a service entity for the current SDK, or its process. Every entity must have some relation to the current SDK.
+- **Resource Coordinator (new)**: A component responsible for taking Resource and Entity detectors and doing the following:
+  - Constructing a Resource for the SDK from detectors.
+  - Dealing with conflicts between detectors.
+  - Providing SDK-internal access to detected Resources for reporting via Log signal on configured LogProviders.
+  - *(new) Managing Entity changes during SDK lifetime, specifically dealing with entities that have lifetimes shorter than the SDK*
+
+#### Resource Coordinator
+
+The SDK Resource Coordinator is responsible for running all configured Resource and Entity Detectors.  There will be some (user-controlled, otel default) priority order to these.
+
+- The Resource Coordinator will detect conflicts in Entity of the same type being discovered and choose one to use.
+- When using Entity Detectors and Resource detectors together, the following merge rules will be used:
+  - Entity merging will occur first resulting in an "Entity Merged" Resource (See [algorithm here](#entity-merging-and-resource)).
+  - Resource detectors otherwise follow existing merge semantics.
+    - The Specification merge rules will be updated to account for violations prevalent in ALL implementation of resource detection.
+    - Specifically: This means the [rules around merging Resource across schema-url will be dropped](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md#merge).  Instead only conflicting attributes will be dropped.
+    - SchemaURL on Resource will need to be deprecated with entity-specific schema-url replacing it. SDKs will no longer fill out SchemaURL on Resource. Additionally, as no (non-service) Resource semantic conventions have ever stabilized, SchemaURL usage on Resource cannot be in stable components of OpenTelemetry.  Given prevalent concerns of implementations around Resource merge specification, we suspect impact of this deprecation to be minimal.
+  - An OOTB ["Env Variable Entity Detector"](#environment-variable-detector) will be specified and provided vs. requiring SDK wide ENV variables for resource detection.
+- *Additionally, Resource Coordinator would be responsible for understanding Entity lifecycle events, for Entities whose lifetimes do not match or exceed the SDK's own lifetime (e.g. browser session).*
+
+#### Entity Detector
+
+The Entity detector in the SDK is responsible for detecting possible entities that could identify the SDK.  For Example, if the SDK is running in a kubernetes pod, it may provide an Entity for that pod.   SDK Entity Detectors are only required to provide identifying attributes, but may provide descriptive attributes to ensure combined Resource contains similar attributes as today's SDK.
```

Comment:
I think the document introduces the concept of descriptive and identifying attributes and it is done here. As this is a key concept, two more sentences should be spent here explaining the general concept *and their impact on equivalence on attribute sets*.

Reply:
Agreed - We've discussed those concepts a bunch in the past, but they're not formalized. I'll look into how to better outline them in other sections.

Reply:
The descriptive and identifying attributes were introduced in [OTEP0256](https://github.com/open-telemetry/oteps/blob/main/text/entities/0256-entities-data-model.md). See the table in "Data Model" section.

Reply:
Given previous otep is (now) linked immediately in this document, I'm closing this as resolved.